docs/source/user-guide/mission-modeler-guide/activity-types/effect-model.md

```diff
@@ -0,0 +1,128 @@
+
+## Activity Decomposition
+In Merlin mission models, decomposition of an activity is not an independent method, rather it is defined within the effect model by means of invoking child activities. These activities can be invoked using the `call()` method, where the rest of the effect model waits for the child activity to complete; or using the `spawn()` method, where the effect model continues to execute without waiting for the child activity to complete. This method allows any arbitrary serial and parallel arrangement of child activities. This approach replaces duration estimate based wait calls with event based waits. Hence, this allows for not keeping track of estimated durations of activities, while also improving the readability of the activity procedure as a linear sequence of events.
+
+## Computed Attributes
+Upon the termination of an Activity, the effect model can optionally log some information that will be included with the simulation results. This information has no effect on the current simulation, but can be used by downstream tools that process the simulation results.
+
+To specify the information that will be logged, the mission modeler must change the return type of the method marked with the @EffectModel annotation.
+
+Let's consider the `RunHeater` example from above:
+```java
+@ActivityType("RunHeater")
+public final class RunHeater {
+    private static final int energyConsumptionRate = 1000;
+
+    @Parameter
+    public long durationInSeconds;
+
+    @Validation("duration must be positive")
+    @Validation.Subject({"durationInSeconds"})
+    public boolean validateDuration() {
+        return durationInSeconds > 0;
+    }
+
+    @EffectModel
+    public void run(final Mission mission) {
+        spawn(new PowerOnHeater());
+
+        final double totalEnergyUsed = durationInSeconds * energyConsumptionRate;
+        mission.batteryCapacity.use(totalEnergyUsed);
+
+        delay(durationInSeconds, Duration.SECONDS);
+
+        call(new PowerOffHeater());
+    }
+}
+```
+The effect model return type is "void", which means no useful information will be logged. Let's change that to `String` and add a log message:
+```java
+@EffectModel
+public String run(final Mission mission) {
+    spawn(new PowerOnHeater());
+
+    final double totalEnergyUsed = durationInSeconds * energyConsumptionRate;
+    mission.batteryCapacity.use(totalEnergyUsed);
+
+    delay(durationInSeconds, Duration.SECONDS);
+
+    call(new PowerOffHeater());
+
+    return "This is a log message!"
+}
+```
+Computed attributes are not limited to strings - they can be any type that merlin knows how to serialize. See [ValueMappers](https://github.com/NASA-AMMOS/aerie/wiki/Activity-Mappers#value-mappers) to learn about what types are supported out of the box, and how to define custom serializers for your own types.
```

Comment:
Link needs to be updated: `See [ValueMappers](https://github.com/NASA-AMMOS/aerie/wiki/Activity-Mappers#value-mappers)`